extensions/amazon-aws/package.json

```diff
@@ -81,6 +81,12 @@
       "description": "List and open stacks, resources and exports",
       "mode": "view"
     },
+    {
+      "name": "glue-jobs",
+      "title": "Glue Jobs",
```

Comment:
At the moment, this command only shows all glue jobs as a list.
With your suggestion, should be present a list of all Glue services, after selecting the "Glue" command?
e.g.
- AWS Lambda (just mentioned here as a reference)
|--> Functions as List
- AWS Glue
|-- Jobs
|  |--> Jobs as List
|-- Data Catalog
|  |-- Databases as List
|  |  |--> Tables as List
...

In my opinion, this seems laborious. In comparison I would suggest:
- AWS Lambda (just mentioned here as a reference)
|--> Functions as List
- Glue Jobs
|--> Jobs as List
- Glue Catalog
|-- Databases
|  |--> Tables as List
|-- Crawlers
|  |--> Crawlers as List

This shrinks the hierarchy tree by one. Maybe Glue is a special service, which in fact is a wrapper for other standalone services.

Reply:
Oh yeah you're right, Glue is kind of a wrapper service. I usually look at the Actions to see what kind of APIs it supports. I would say still go with `Glue` b/c we can always structure the `ActionPanel.Submenu` with sections for different resource types per-service. Here is a mock up:

<img width="356" alt="Screenshot 2024-06-19 at 4 45 07 PM" src="https://github.com/raycast/extensions/assets/42366677/a6902c92-a507-4025-93bc-648d8290cfc8">

It is always a 2-way door decision and we can separate it into 2 commands later in case it gets complicated but let's start with one command first. We can always demonstrate through description the resources we can manage in the command.